src/components/Components/ContactUsFormComponent/ContactUsFormComponent.tsx

```diff
@@ -0,0 +1,70 @@
+import { Form, Button, Input, message } from "antd";
+import TextArea from "antd/es/input/TextArea";
+
+import styles from "./ContactUsFormComponent.module.css";
+
+export const ContactUsFormComponent = () => {
+  const [form] = Form.useForm();
+
+  const onFinish = (values: unknown) => {
+    void message.info(JSON.stringify(values));
+  };
```

Comment:
onFinish could init business logic
so, change Logic that can receive onFinish logic from props

onFinish={props.onFinish}

Reply:
I've updated the code to initialize some business logic in onFinish and to pass onFinish from props. Changes have been applied to this and all subsequent branches pushed today